### PR TITLE
Adds Admin teleporter

### PR DESCRIPTION
**Pull request by Myzumi**
## About the PR

This adds in a admin version of the handheld teleporter, which works across maps and dosent make a random teleport when only one portal 
## Why / Balance

It has been requested by admins, so here it is
## How to test

- Load Frontier
- Spawn in the admeme hand teleporter
- Create Portals across maps
- Get teleported

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- add: Added Admeme Hand teleporter

### CODE REVIEW

File: Resources/Prototypes/_NF/Objects/Devices/admin_hand_teleporter.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+- type: entity
+  id: AdminHandTeleporter
+  parent: BaseItem
+  name: hand teleporter [admeme]
```
Comment (dvir001) [inline Resources/Prototypes/_NF/Objects/Devices/admin_hand_teleporter.yml:4]: Use suffix

Also please use main existing items as parents you dont need this 200 lines just for

```
  - type: Portal
    canTeleportToOtherMaps: true
```

File: Resources/Prototypes/_NF/Entities/Effects/portal.yml [hunk 2]
```diff
@@ -1,56 +1,16 @@
 - type: entity
-  id: BaseAdminPortal
+  id: BasePortalAdmin
+  perent: BasePortal
```
Comment (Leander-0) [inline Resources/Prototypes/_NF/Entities/Effects/portal.yml:3]: Giant blunder

Comment (dvir001) [reply Resources/Prototypes/_NF/Entities/Effects/portal.yml:3]: I aint done here

Comment (dvir001) [reply Resources/Prototypes/_NF/Entities/Effects/portal.yml:3]: @Leander-0 Now im done
Saved like 60 lines

**Review by dvir001 — APPROVED** [review]

File: Resources/Prototypes/_NF/Entities/Effects/portal.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+- type: entity
+  id: BasePortalAdmin
+  perent: BasePortal
```
Comment (whatston3) [inline Resources/Prototypes/_NF/Entities/Effects/portal.yml:3]: Typo here, should be `parent`.  Hotfix inbound?

Comment (dvir001) [reply Resources/Prototypes/_NF/Entities/Effects/portal.yml:3]: Go for it

**Pull request merged**